[DO NOT MERGE] Documentation update review (#172)
canonical/kafka-operator: docs/index.md

> ## Charmed Kafka Documentation

Reviewer: To keep a consistent home page structure across charms, order would be:
1. Title
2. Description
3. "In this documentation"
4. "Project and community"
5. License

Charmed Kafka is an open-source operator that makes it easier to operate Apache Kafka with built-in support for enterprise features. 

> Charmed Kafka is build on top of [Juju](https://juju.is/) and it simplifies the deployment, scaling, design and management of Apache Kafka in production in a reliable way. In addition, you can use the operator to manage your Kafka clusters with automation capabilities. It also offers replication, TLS, password rotation, easy-to-use application integration, and monitoring.

Reviewer: ```suggestion
Charmed Kafka is built on top of [Juju](https://juju.is/) and reliably simplifies the deployment, scaling, design, and management of Apache Kafka in production. Additionally, you can use the charm to manage your Kafka clusters with automation capabilities. It also offers replication, TLS, password rotation, easy-to-use application integration, and monitoring.
```

> Charmed Kafka operates Apache Kafka on physical, Virtual Machines (VM) and other wide range of cloud and cloud-like environments, including AWS, Azure, OpenStack and VMWare. 

Reviewer 2: ```suggestion
Charmed Kafka operates Apache Kafka on physical systems, Virtual Machines (VM) and a wide range of cloud and cloud-like environments, including AWS, Azure, OpenStack and VMWare. 
```

> Charmed Kafka is a solution designed and developed for helping organizations and 

Reviewer 2: ```suggestion
Charmed Kafka is a solution designed and developed for helping ops teams and 
```

> DB administrators automating Apache Kafka operations management from day 0 to day 2, across multiple cloud environments and substrates.

Reviewer 2: ```suggestion
administrators to automate Apache Kafka operations from day 0 to day 2, across multiple cloud environments and platforms.
```

> Aside from a VM operator, Charmed Kafka K8s provides support for Kubernetes environments. For more information, please refer to the Charmed Kafka K8s documentation [here](/t/charmed-kafka-k8s-documentation/10296).

Reviewer: ```suggestion
[note]
Canonical has also developed the [Charmed Kafka K8s operator]((/t/charmed-kafka-k8s-documentation/10296)) to support Kafka in Kubernetes environments.
[/note]
```

> Charmed Kafka is developed and supported by [Canonical](https://ubuntu.com/), as part of its commitment to 

Reviewer 2: ```suggestion
Charmed Kafka is developed and supported by [Canonical](https://canonical.com/), as part of its commitment to 
```
